public/locales/en/common.json

```diff
@@ -146,5 +146,21 @@
   "taxIdentificationNumber": "Tax Identification Number",
   "tinRequired": "Tax Identification Number is Required",
   "tinText": "TIN:",
-  "returnTo": "Return to"
-}
\ No newline at end of file
+  "returnTo": "Return to",
+  "donateWithPlanetCash": "Donate with PlanetCash",
+  "donationOnBehalfText": "This donation is on behalf of someone",
+  "donationOnBehalfOf": "This donation is on behalf of {{firstName}} {{lastName}}",
+  "usePlanetCash": "Use PlanetCash for this donation",
+  "balance": "Balance",
+  "credit": "Credit",
+  "addBalance": "Add Balance",
+  "onBehalfOf": "On behalf of",
+  "inValidField": "{{fieldName}} is invalid",
```

Comment:
there should be existing text for this "inValidField"

Reply:
There isn't one.